content/homepage/index.fr.md

```diff
@@ -0,0 +1,22 @@
+---
+title: page d'accueil
+displayTitle: Exécuter JavaScript partout.
+subTitle: 'Node.js est un environnement d'exécution JavaScript gratuit, ouvert et multiplateforme qui permet aux développeurs d'écrire des outils en ligne de commande et des scripts côté serveur en dehors d'un navigateur.'
```

Comment:
Problem here is that you’re using single quote, but one of yours words also has single quote. Hence why I mentioned to use double quotes 👀


Also, please do not change the `title` field, it needs to be the original word in English